CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [0.0.next] - 2017-06-02
+### Changed
+- java-heap-pcnt.sh and metrics-java-heap-graphite.sh: removed extra column from 'UsedHeap'
```

Comment:
I'd like a little more info for others as to why this is breaking. This is my 2cents on what it should say:
```
- java-heap-pcnt.sh and metrics-java-heap-graphite.sh removed some columns that were used by java 7 that we are dropping in favor of supporting java 8 properly.
```